lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/ConfigurationService.java

```diff
@@ -93,8 +110,12 @@ public ConfigurationService(
         this.dcsEndpointUri = paramProvider.get(getParameterName("dcsEndpoint"));
         this.documentCheckResultTableName =
                 paramProvider.get(getParameterName("DocumentCheckResultTableName"));
+        this.useLegacy = Boolean.parseBoolean(paramProvider.get(getParameterName("useLegacy")));
```

Comment:
Should this be under the feature toggles bracket?

Reply:
Definitely good shout